packages/flutter/lib/src/services/platform_channel.dart

```diff
@@ -98,6 +98,9 @@ class BasicMessageChannel<T> {
   }
 }
 
+Expando<Object> _methodChannelHandlers = Expando<Object>();
+Expando<Object> _methodChannelMockHandlers = Expando<Object>();
```

Comment:
This is the only way I can think of to keep the MethodChannel class const constructable. Making it non-const is probably pretty breaking, and we can't otherwise compare the handler functions properly.

Reply:
I would be open to guarding these for release mode, but wouldn't want them to be assert guarded since we might want to use this functionality for e2e tests in profile mode.

Reply:
We are also running e2e tests on release mode. For example, sometimes a functionally doesn't work in release mode.